### PR TITLE
Support for deferred parsing of HTML event handlers.

### PR DESCRIPTION
**Pull request by pleath**
Support in the ChakraCore repo for deferred parsing of HTML event handlers.

### CODE REVIEW

**Comment by pleath** [comment]
@curtisman @Yongqu

File: lib/Runtime/Base/FunctionBody.cpp [hunk 1]
```diff
@@ -3562,7 +3557,7 @@ namespace Js
         newFunctionBody->m_isFromNativeCodeModule = this->m_isFromNativeCodeModule;
     }
 
-    FunctionBody *
+    ParseableFunctionInfo *
```
Comment (Yongqu) [inline lib/Runtime/Base/FunctionBody.cpp]: It should be safe to get rid of the clone code now, replace with assert. I'll see if I have time to get rid of the caller side of the code.

**Comment by Yongqu** [comment]
otherwise Looks good to me.

File: lib/Runtime/Base/FunctionBody.cpp (hunk 1 above)
Comment (pleath) [inline lib/Runtime/Base/FunctionBody.cpp]: How about if we do that as a separate change?

-- Paul
________________________________

**Comment by Yongqu** [comment]
agree don't need to make that change now. LGTM.

**Pull request merged**